--- 
Header header
 string port
 float64 temperature
 bool success